hack/templates/values/values.template

```diff
@@ -122,35 +139,46 @@ sriov-network-operator:
       #       ...
       #      -----END EC PRIVATE KEY-----
 
-  # Image URIs for sriov-network-operator components
+  # Image URIs for sriov-network-operator components.
   images:
     operator: {{ .SriovNetworkOperator.Repository }}/{{ .SriovNetworkOperator.Image }}:{{ .SriovNetworkOperator.Version }}
     sriovConfigDaemon: {{ .SriovConfigDaemon.Repository }}/{{ .SriovConfigDaemon.Image }}:{{ .SriovConfigDaemon.Version }}
+    # -- For ARM-based deployments, it is recommended to use the
+    # ``ghcr.io/k8snetworkplumbingwg/sriov-cni:v2.8.0-arm64`` image.
```

Comment:
We should not have specific versions in the template.
@e0ne is this doc still relevant for next version or we have multi-arch images already?

Reply:
Good point. Waiting for @e0ne answer. If we still need that comment, I'll remove the tag.

Reply:
its all muti-arch today.

sriov. ib-sriov cni , device plugin ,